pkg/scannerv4/mappers/mappers.go

```diff
@@ -383,20 +378,29 @@ func toProtoV4VulnerabilitiesMap(ctx context.Context, vulns map[string]*claircor
 		if vulnerabilities == nil {
 			vulnerabilities = make(map[string]*v4.VulnerabilityReport_Vulnerability, len(vulns))
 		}
-		vulnerabilities[k] = &v4.VulnerabilityReport_Vulnerability{
+		vuln := &v4.VulnerabilityReport_Vulnerability{
 			Id:                 v.ID,
 			Name:               name,
 			Description:        description,
 			Issued:             issued,
 			Link:               v.Links,
-			Severity:           sev.severity,
+			Severity:           primarySev.severity,
```

Comment:
Do we need to consider severity's per source as well? In the past (not sure if doing the same for Scanner V4) we had [converted CVSS scores](https://github.com/stackrox/scanner/blob/49768d3baa6729cfe2a68554c06c1159f1a8b754/pkg/cvss/util.go#L15) to a 'severity' if not provided by the source directly.  It seems that NVD may have a different severity vs. redhat vs. other.

Reply:
Currently (though this may change in the future), only CVSS scores from specific sources are required, and the NVD score is not derived from severity.